compiler/ballerina-lang/src/test/java/io/ballerina/projects/ProjectUtilsTests.java

```diff
@@ -52,23 +53,29 @@ public void setUp() throws IOException {
     @Test
     public void testReadBuildJson() {
         Path buildFilePath = PROJECT_UTILS_RESOURCES.resolve(ProjectConstants.BUILD_FILE);
-        BuildJson buildJson = ProjectUtils.readBuildJson(buildFilePath);
-        Assert.assertEquals(buildJson.lastBuildTime(), 1629359520);
-        Assert.assertEquals(buildJson.lastUpdateTime(), 1629259520);
+        try {
+            BuildJson buildJson = ProjectUtils.readBuildJson(buildFilePath);
+            Assert.assertEquals(buildJson.lastBuildTime(), 1629359520);
+            Assert.assertEquals(buildJson.lastUpdateTime(), 1629259520);
+        } catch (Exception e) {
+            Assert.fail("Reading Build Json failed");
+        }
     }
 
-    @Test(expectedExceptions = ProjectException.class,
-            expectedExceptionsMessageRegExp = "Failed to read the 'build' file")
+    @Test()
     public void testReadBuildJsonForNonExistingBuildFile() {
         Path buildFilePath = PROJECT_UTILS_RESOURCES.resolve("xyz").resolve(ProjectConstants.BUILD_FILE);
-        ProjectUtils.readBuildJson(buildFilePath);
+        Assert.assertThrows(IOException.class, () -> {
```

Comment:
Can we assert the exception message here?

Reply:
After changing the implementation, the function does not throw any particular exceptions with set messages. ie. the exception message can vary, so we decided to assert the exception type directly